modules/apps/foundation/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/site/facet/portlet/ScopeFacetBuilder.java

```diff
@@ -49,6 +49,10 @@ public Facet build() {
 		return facet;
 	}
 
+	public static String[] getSelectedSites() {
```

Comment:
Ran into errors regarding static methods vs non-static methods. Created a getter method to attempt to returned the needed data to the MVCActionCommand java class.